src/Propulsion.MessageDb/Readme.md

```diff
@@ -0,0 +1,49 @@
+# Propulsion.MessageDb
+
+This project houses a Propulsion source for [MessageDb](http://docs.eventide-project.org/user-guide/message-db/).
+
+## Quickstart
+
+The smallest possible sample looks like this, it is intended to give an overview of how the different pieces relate. 
+For a more production ready example to take a look at [jets' templates](https://github.com/jet/dotnet-templates)
+
+```fsharp
+let quickStart log stats trancheIds handle = async {
+    // The group is used as a key to store and retrieve checkpoints 
+    let groupName = "MyGroup"
+    // The checkpoint store will receive the highest version
+    // that has been handled and flushes it to the
+    // table on an interval
+    let checkpoints = ReaderCheckpoint.CheckpointStore("Host=localhost; Port=5433; Username=postgres; Password=postgres", "public", groupName, TimeSpan.FromSeconds 10)
+    // Creates the checkpoint table in the schema
+    // You can also create this manually
+    do! checkpoints.CreateSchemaIfNotExists()
+    
+    let client = MessageDbCategoryClient("Host=localhost; Database=message_store; Port=5433; Username=message_store; Password=;")
+    let maxReadAhead = 100
+    let maxConcurrentStreams = 2
+    use sink = 
+      Propulsion.Streams.Default.Config.Start(
+        log, maxReadAhead, maxConcurrentStreams, 
+        handle, stats, TimeSpan.FromMinutes 1)
+        
+    use src = 
+      MessageDbSource(
+        log, statsInterval = TimeSpan.FromMinutes 1,
+        client, batchSize = 1000, 
+        // Controls the time to wait once fully caught up
+        // before requesting a new batch of events
+        tailSleepInterval = TimeSpan.FromMilliseconds 100,
+        checkpoints, sink,
+        // tranche is equivalent to a message-db category
+        trancheIds
```

Comment:
this and the underlying impl should call this categories and they should be a string array (maybe primary ctor can use trancheids and the real ctor can do the Array.map )

Reply:
Yeah, good point.